README.md

```diff
@@ -24,6 +24,20 @@ SolidusSupport::Migration[5.0] # same as `ActiveRecord::Migration[5.0]`
 
 There's no reason to use `SolidusSupport::Migration[5.0]` over `ActiveRecord::Migration[5.0]`, but it is provided.
 
+### Testing Helpers
+
+This gem provides some useful helpers for RSpec to setup an extension's test
+environment easily.
+
+Into your extension's spec/spec_helper.rb:
+
+```ruby
+require "solidus_support/extension/feature_helper"
+```
+
+`feature_helper` will also require `rails_helper` and `spec_helper`,
+which can even be used standalone if needed.
```

Comment:
We should also mention that the other helpers are requirable as well. And explain which helper is meant for what tests.

Reply:
I thought this part would explain that:

> which can even be used standalone if needed

but I was probably wrong. I'll try to improve it further explaining what each helper is needed for.